course_discovery/apps/api/v1/tests/test_views/test_course_runs.py

```diff
@@ -279,6 +295,27 @@ def test_contains_multiple_course_runs(self):
             }
         )
 
+    def test_contains_multiple_course_runs_organization_filter(self):
+        qs = urllib.parse.urlencode({
+            'query': 'id:course*',
+            'course_run_ids': '{},{},{}'.format(self.course_run.key, self.course_run_2.key, 'abc')
```

Comment:
1. Instead of `abc` use some proper values, e.g. `spring_2020`
2. use key base formatting
3. consider making course run id's similar to default edX course run id `course-v1:edX+DemoX+Demo_Course`

Reply:
Done